http/org_service.go

```diff
@@ -440,10 +440,13 @@ func (h *OrgHandler) handleGetSecrets(w http.ResponseWriter, r *http.Request) {
 	}
 
 	ks, err := h.SecretService.GetSecretKeys(ctx, req.orgID)
-	if err != nil {
+	if err != nil && influxdb.ErrorCode(err) != influxdb.ENotFound {
 		h.HandleHTTPError(ctx, err, w)
 		return
 	}
+	if ks == nil {
+		ks = []string{}
```

Comment:
Instead of doing the markup here, why dunt we just simply return an empty string in the kv layer?

Reply:
Yeah, you are right